mfa.tf

```diff
@@ -0,0 +1,17 @@
+data "aws_caller_identity" "current" {}
+
+resource "aws_iam_group" "test-account-module" {
```

Comment:
module name should not be like "test-account-*", this seems from some example, the name of module should contain "_" as word separator instead of "-", have the name similar to "enforce_mfa_iam_group"